app/src/main/java/com/kickstarter/libs/ActivityRequestCodes.java

```diff
@@ -8,4 +8,5 @@ private ActivityRequestCodes() {}
   public final static int CHECKOUT_ACTIVITY_WALLET_CHANGE_REQUEST = 592_11;
   public final static int CHECKOUT_ACTIVITY_WALLET_OBTAINED_FULL = 592_12;
   public final static int SAVE_NEW_PAYMENT_METHOD = 592_13;
+  public final static int SHOW_REWARDS = 592_14;
```

Comment:
What's a request code?

Reply:
If you want to get a result back from a screen you started, you need to pass in a request code `Int`. https://developer.android.com/training/basics/intents/result#StartActivity. So if I want the `ProjectActivity` to know that the `CampaignDetailsActivity`'s pledge action button was clicked, I need to use a unique request code.